example_configs/spark.yml

```diff
@@ -1,40 +1,53 @@
+---
+lowercaseOutputName: true
```

Comment:
I wouldn't add these, I'm starting to feel that munging names causes more confusion than it does clarity

Reply:
Delete both `lowercaseOutputName` and `attrNameSnakeCase` or just the former?

Reply:
Both please, they weren't here before.